### ISSUE TITLE
Support for game saves

### ISSUE THREAD

**Issue by michail-nikolaev** [michail-nikolaev/task-force-arma-3-radio#867]

**Comment by dedmen** [comment]
I don't think we need to do anything here. Our namespace will be saved. Thus our radio settings are saved. And everyone keeps the radio in their inventory that they had before.
